src/Middleware/src/Headstart.Common/Assets/english-translations.json

```diff
@@ -488,13 +491,13 @@
   "AUTHENTICATION": {
     "FORGOT_PASSWORD": {
       "HEADER": "Forgot Password",
-      "SUBHEADER": "Please provide either your email or username",
       "LOGIN": "Return to Login",
-      "SEND_EMAIL": "Send Email"
+      "SEND_EMAIL": "Send Email",
+      "SUBHEADER": "Enter Email/Username to reset password"
     },
     "LOGIN": {
       "ACCOUNT_LOGIN": "Account Login",
-      "LOGIN": "Login",
+      "LOGIN": "Login with Anytime Fitness Account",
```

Comment:
Is this just something that has yet to be axed from Headstart? Looks like this would be for SEB specific SSO?

Reply:
ope, good catch. axed.